server/server.js

```diff
@@ -1,15 +1,170 @@
 const express = require("express");
+const cors = require("cors");
+const pool = require("./db");
+const { v4: uuidv4 } = require("uuid");
+const { body, validationResult } = require("express-validator");
+require("dotenv").config();
+
 const app = express();
-const port = process.env.PORT || 5000;
+app.use(cors());
+app.use(express.json());
 
-app.listen(port, () => console.log(`Listening on port ${port}`));
+// Get all videos
+app.get("/", async (req, res) => {
+  const { order, search } = req.query;
+  try {
+    const allVideos = await pool.query("SELECT * FROM videos");
+    let filteredVideos = [...allVideos.rows];
+    // Apply search filter if search query is provided
+    if (search) {
+      filteredVideos = filteredVideos.filter((video) =>
+        video.title.toLowerCase().includes(search.toLowerCase())
+      );
+    }
+    //Apply order filter
+    if (order === "desc") {
```

Comment:
I think you're sorting on both the client side and the server side - do you need to do both? What are the advantages/disadvantages of doing it in each place?